modules/animation/include/modules/animation/datastructures/valuekeyframesequence.h

```diff
@@ -174,7 +177,7 @@ void KeyframeSequenceTyped<Key>::operator()(Seconds from, Seconds to, value_type
     if (interpolation_) {
         (*interpolation_)(this->keyframes_, from, to, easing_, out);
     } else {
-        out = this->keyframes_.front()->getValue();
+        IVW_ASSERT(true, "Interpolation must be set before animating")
```

Comment:
If this is the case why not enforce that there always is an interpolation, if it is mandatory it should be passed in or created during construction, not later on.

Reply:
Perhaps we should. Ideally, we would then remove the constructors without interpolation. However, this is not possible, or at least I get compile errors then. We could throw exceptions in those constructors. That would mean that one needs to at least now a default interpolation during construction, which might be ok.

Reply:
I have enforced it now. Took a while to figure out the needed template magic :D